### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by tobihy**
Let's make the User Guide more reader-focussed and reader-friendly, as well as fix some minor grammatical errors and improve on sentence structures.

@tsumian Do let me know if there's any edits that you disagree with, thanks!

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -343,8 +363,7 @@ Examples:
 
 ### List all pets : `list`
 
-After using all of those commands, you may want to retrieve the initial pet list with all the pets. This feature just
-shows a list of all pets in pet list.
+This feature lists out all pets, helpful for when you wish to look through all pets in WoofAreYou.
 
 Format:`list`
```
Comment (soloplxya) [inline docs/UserGuide.md:368]: Possible to put a space after the colon for consistency?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -5,17 +5,24 @@ title: User Guide
 * Table of Contents
 {:toc}
 --------------------------------------------------------------------------------------------------------------------
+## Motivation
 
+As a pet daycare owner, it takes a lot of effort to remember pets by their names,
+as well as remember their owner's names, phone numbers and addresses. In addition, some pets may have special dietary requirements and
+vet appointments that you have to take note off. With WoofAreYou, these things can be taken care
```
Comment (soloplxya) [inline docs/UserGuide.md:12]: Possible to replace `take note off` with `take note of`?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -5,17 +5,24 @@ title: User Guide
...
+of easily, so that you may focus on the things that **truly** matter: taking good care of the pets and ensuring their safety.
+--------------------------------------------------------------------------------------------------------------------
 # WoofAreYou
 
-WoofAreYou is a desktop application for pet daycare owners to handle the administrative information of their clients. 
-WoofAreYou is faster than any traditional platform such as Microsoft Excel or Google Sheets when you want to consolidate
+WoofAreYou is a desktop application for pet daycare owners like you to handle the administrative information of your clients. 
+WoofAreYou is faster than traditional platforms, such as Microsoft Excel or Google Sheets, when you want to consolidate
```
Comment (soloplxya) [inline docs/UserGuide.md:18]: I was thinking maybe can say `many traditional platforms`, to make our app sound cooler. Not sure though?

**Review by soloplxya — COMMENTED** [review]
Some grammar nits! Other than that, LGTM! 👍

File: docs/UserGuide.md [hunk 4]
```diff
@@ -230,35 +239,33 @@ Examples:
...
 * The index refers to the index number shown in the current list of pets.
 * The index **must be a positive integer** 1, 2, 3, …​
-* Date and time of appointment should be entered together with `dt/` prefix.
+* Date and time of the appointment should be entered after the `dt/` prefix.
 * Date and time should strictly follow `dd-MM-yyyy HH:mm` format.
-* Location of appointment should be entered with `at/` prefix.
+* Location of appointment should be entered after the `at/` prefix.
 * Whitespaces, special characters and alphanumeric characters are allowed for location.
-* If both `date/` and `at/` are not present, `app` will be deemed invalid.
+* Both `date/` and `at/` are **mandatory** fields.
```
Comment (tsumian) [inline docs/UserGuide.md:265]:
```suggestion
* Both `dt/` and `at/` are **mandatory** fields.
```
Did not notice this initially when I did it

**Review by tsumian — COMMENTED** [review]
Just one minor change needed

File: docs/UserGuide.md (hunk 4 above)
Comment (tobihy) [reply docs/UserGuide.md:265]: Thanks for pointing it out!

**Pull request merged**